source/module_orbital/ORB_table_phi.cpp

```diff
@@ -910,3 +916,18 @@ void ORB_table_phi::init_Table_Spherical_Bessel (
 //	OUT(GlobalV::ofs_running,"Rmesh",Rmesh);
 	ModuleBase::Memory::record ("ORB_table_phi", "Jl(x)", (Lmax_used+1) * this->kmesh * this->Rmesh, "double");
 }
+
+void ORB_table_phi::plot_table(
+	const std::string filename,
+	const int rmesh,
+	double* &column)
```

Comment:
this type is confused to me , why do you use :
"
void ORB_table_phi::plot_table(
	const std::string &filename,
	const int &rmesh,
	const double* column)
"